src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +136,27 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testAesdefaultmodepadding() throws InvalidParametersException {
+        String msg = "Test AES Default";
+        String decrptedMsg;
+        byte[] decrypted;
+        decrptedMsg = StringHelper.encrypt(msg);
+
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.DECRYPT_MODE, sks);
+            decrypted = cipher.doFinal(StringHelper.hexStringToByteArray(decrptedMsg));
```

Comment:
@Neurrone  I think earlier way of encrypt with "AES/ECB/PKCS5Padding" and decrypt with "AES" would suffice as i am unable to get the mode and padding through getParameters.
So in which way to proceed.